packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -0,0 +1,292 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+import 'button.dart';
+
+// Padding around the line at the edge of the text selection that has 0 width and
+// the height of the text font.
+const double _selectionHandlesPadding = 18.0;
+const double _kToolbarScreenPadding = 8.0; // pixels
+const double _kToolbarHeight = 36.0;
+
+const Color _selectionToolbarBackgroundBlack = const Color(0xFF2E2E2E);
+const Color _selectionToolbarDividerGray = const Color(0xFFB9B9B9);
+const Color _selectionHandlesBlue = const Color(0xFF146DDE);
```

Comment:
nit: Having a description of the color in the name of color constants seems a bit odd to me.  "Color" instead of "Black/Gray/Blue" would concentrate the knowledge of what these colors actually are to a single place instead of everywhere the constants are use

You can see this in other places in Flutter, e.g. `_kActiveTickColor`, `_kDisabledBackground`